src/pages/EditMonitor.vue

```diff
@@ -899,6 +902,28 @@
                                     </div>
                                 </template>
                             </template>
+
+                            <!-- TLS Options -->
+                            <template v-if="monitor.type === 'port-tls'">
+                                <h2 class="mt-5 mb-2">{{ $t("TLS Options") }}</h2>
+
+                                <!-- Use STARTTLS -->
+                                <div class="my-3 form-check">
+                                    <input id="start-tls" v-model="monitor.startTls" class="form-check-input" type="checkbox">
+                                    <label class="form-check-label" for="start-tls">
+                                        {{ $t("Use STARTTLS") }}
+                                    </label>
+                                </div>
+
+                                <!-- Request -->
+                                <div class="my-3">
+                                    <label for="request" class="form-label">{{ $t("Request") }}</label>
+                                    <textarea id="request" v-model="monitor.request" class="form-control"></textarea>
+                                </div>
+                                <div class="form-text">
+                                    {{ $t("requestDescription") }}
```

Comment:
Okay..
I would not know what should be put here. Could we include an example or link to an RFC as relevant docs?

Reply:
My plan is to have a) links to the relevant RFCs and b) a few buttons for common protocols that will insert common requests and keywords automatically. It turns out (not at all surprisingly) that SMTP, POP3, IMAP4 all use different commands/responses, so I'll have to add support for custom STARTTLS commands as well. Basically, the UI is going to change quite a bit. Stay tuned.